sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -433,19 +433,21 @@ public void testTableConstraintsWithOrderBy() throws Exception {
     @Test
     public void testDefaultColumns() throws Exception {
         execute("select * from information_schema.columns order by schema_name, table_name");
-        assertEquals(308L, response.rowCount());
+        assertEquals(310L, response.rowCount());
     }
 
     @Test
     public void testColumnsColumns() throws Exception {
         execute("select * from information_schema.columns where schema_name='information_schema' and table_name='columns' order by ordinal_position asc");
-        assertEquals(5, response.rowCount());
+        assertEquals(7, response.rowCount());
         short ordinal = 1;
-        assertArrayEquals(response.rows()[0], new Object[]{"information_schema", "columns", "schema_name", ordinal++, "string"});
-        assertArrayEquals(response.rows()[1], new Object[]{"information_schema", "columns", "table_name", ordinal++, "string"});
-        assertArrayEquals(response.rows()[2], new Object[]{"information_schema", "columns", "column_name", ordinal++, "string"});
-        assertArrayEquals(response.rows()[3], new Object[]{"information_schema", "columns", "ordinal_position", ordinal++, "short"});
-        assertArrayEquals(response.rows()[4], new Object[]{"information_schema", "columns", "data_type", ordinal, "string"});
+        assertThat(response.rows()[0], is($("information_schema", "columns", "schema_name", ordinal++, "string", false, null)));
```

Comment:
pls add a test that shows an actual generated column as soon as possible via DDL